From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Niklas Haas <git@haasn.xyz>
Date: Wed, 17 Jun 2020 16:56:19 +0200
Subject: [PATCH] spirv/glslang: fix build on 8.13.3743

The API change referenced in 5230568 was introduced in between patch
level 3743 and patch level 3763. The current check breaks the tagged
release 8.13.3743 which does not include the change yet.

Bump the check up to 3763 to get around this. In classic glslang style,
this means that there are a few commits which will always fail to
compile no matter what we do... but that's the reality we have to live
with when building against glslang. It's either that or they figure out
how to bump their patch level on breaking API changes.
---
 src/glsl/glslang.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/glsl/glslang.cc b/src/glsl/glslang.cc
index fcd1745..c237541 100644
--- a/src/glsl/glslang.cc
+++ b/src/glsl/glslang.cc
@@ -211,7 +211,7 @@ const TBuiltInResource DefaultTBuiltInResource = {
     /* .maxTaskWorkGroupSizeZ_NV = */ 1,
     /* .maxMeshViewCountNV = */ 4,
 #endif
-#if GLSLANG_PATCH_LEVEL >= 3743
+#if GLSLANG_PATCH_LEVEL >= 3763
     /* .maxDualSourceDrawBuffersEXT = */ 1,
 #endif
 
